src/SelectField.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Box, Text } from '../src'
+
+const parseIsSelected = props => {
+  return `
+    ${props.isSelected
+      ? `background-color: ${props.theme.colors['blue']};`
+      : null}
```

Comment:
Just FYI, styled-components also accepts style objects when you're writing functions like this, which can make the code a little cleaner and avoid the need to nest template literals – the [polished](https://polished.js.org) library does this

Reply:
Great feedback. Took a look at the "polished" library to learn about their pattern. Updated here to a pattern closer to the suggested idea. Please review again.